.github/workflows/test.yaml

```diff
@@ -270,6 +270,11 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version-major }}.${{ matrix.python-version-minor }}
+      - name: Set pyo3 forward compatibility environment variable
```

Comment:
what is this for?

Reply:
Trying to resolve this error: https://github.com/wandb/weave/actions/runs/11578556355/job/32232699701?pr=2757